WORKSPACE

```
@@ -311,55 +311,55 @@ http_archive(
 http_archive(
     name = "nodejs10_amd64",
     build_file = "//nodejs:BUILD.nodejs",
-    sha256 = "aa7e9e1d8abcc169119bf5c56ede515689f2644ccc4d40ca0fc33756a3deb1f7",
-    strip_prefix = "node-v10.22.0-linux-x64/",
+    sha256 = "19cccb78f0881a78051291a50200200a0303649ee84e5489c771d3b4e4bd0e51",
+    strip_prefix = "node-v10.23.0-linux-x64/",
     type = "tar.gz",
-    urls = ["https://nodejs.org/dist/v10.22.0/node-v10.22.0-linux-x64.tar.gz"],
+    urls = ["https://nodejs.org/dist/v10.22.0/node-v10.23.0-linux-x64.tar.gz"],
```

Probably
```suggestion
    urls = ["https://nodejs.org/dist/v10.23.0/node-v10.23.0-linux-x64.tar.gz"],
```
Please fix other URLs too.

---

oops, I fixed other places also - thanks!